i18n/en.i18n.json

```diff
@@ -588,6 +590,7 @@
   "Yes_mute_user" : "Yes, mute user!",
   "Yes_remove_user" : "Yes, remove user!",
   "you_are_in_preview_mode_of" : "You are in preview mode of channel #<strong>__room_name__</strong>",
+  "You_are_logged_as" : "You are logged as",
```

Comment:
I think the correct message should be `You are logged in as` but not sure :stuck_out_tongue: